recipes-kernel/linux/linux-smelt/defconfig

```diff
@@ -1165,8 +1159,8 @@ CONFIG_USB_HSIC_SMSC_HUB=y
 # CONFIG_QCOM_LIQUID_DOCK is not set
 CONFIG_MMI_FACTORY=y
 CONFIG_BQ5105X_CTRL=y
-CONFIG_MOT_UTAG=y
-CONFIG_SSV_GPS=y
+# CONFIG_MOT_UTAG is not set
+# CONFIG_SSV_GPS is not set
```

Comment:
Is this not the integrated GPS in smelt?

Reply:
Nope, only the sport edition has GPS.